docs/developer-quickstart.md

```diff
@@ -115,92 +115,3 @@ for await (const message of await xmtp.conversations.streamAllMessages()) {
 }
 //eg. New message from 0xd8dA6BF26964aF9D7eEd9e03E53415D37: gm
 ```
-
```

Comment:
Just making sure that you want to delete the Troubleshooting and Bots sections? 

If you think they are no longer useful - I trust your judgment! 😀

Reply:
Thanks! We still have the Bot section in the overview. The troubleshoot now is removed and is part of the repo issues.